clamor/gateway/connector.py

```diff
@@ -99,22 +104,22 @@ def format_url(self):
     async def _receive(self):
         message = await self._con.get_message()
         logger.debug("Received message '{}'".format(message))
-        if self.zlib_compressed:
-            # handle zlib compression here
-            pass
-        else:
-            # As there are special cases where zlib-compressed payloads also occur, even
-            # if zlib-stream wasn't specified in the Gateway url, also try to detect them.
-            is_json = message[0] == '{'
-            is_etf = message[0] == 131
-            if not is_json and not is_etf:
-                message = zlib.decompress(message, 15, self.TEN_MEGABYTES).decode('utf-8')
+
```

Comment:
Would be nice to have extra tests for uncompressed connection and compressed connection each

Reply:
Fixed, the test_gateway_connection file tests for both now.